Assets/Scripts/Source/Game/Combat/CombatManager.cs

```diff
@@ -21,14 +37,22 @@ public virtual void Init(CharacterManager characterManager, TurnManager turnMana
 
     public void PlayerAttack()
     {
+        if (!turnManager.ShouldWaitForPlayerAction(characterManager.GetActivePlayer()))
+            return;
```

Comment:
Should we log a warning here to indicate we're skipping the player's action due to an invalid state?

Reply:
Nah because then we would have to add brackets and that would add .00000001s to compile time.